### DISCUSSION TITLE
google coral accelerator + HAAS integration

### DISCUSSION THREAD

**Discussion by heapxor** [blakeblackshear/frigate#1782] [General]
Hello,
i read thru the documentation but its still not clear how can I run Frigate on google coral accelerator and using/installing it as HASS plugin with my HASS running on other computer - odroid xu4; is that possible? 
 
https://github.com/blakeblackshear/frigate-hass-integration 

Thank you

**Comment by haggercody** [comment]
youd also need either the frigate add on installed in HAAS or frigate running in the docker.

**Reply by heapxor** [reply]
@haggercody not sure ... so there are only 2 options...
1) run frigate addon in HAAS on some PC
2) run frigate in the docker running on Google coral hw?

ie i cant run frigate UI/etc... and computer power ie some AI engine will be running on google coral and somehow synchronized with frigate UI running as addon in HAAS?

thanks

**Reply by blakeblackshear** [reply]
Frigate does not run on the Coral. The Coral is not a computer. It is a usb device that you plug into a computer with a specialized processor. It is kind of like a GPU.

**Reply by heapxor** [reply]
@blakeblackshear thanks, was my fault ... now i can see that the best way would be to run HASS on rpi4 then frigate via frigate-hass-integration and plug via usb https://coral.ai/products/accelerator to the RPi4; so then it should be fine. Hopefully?

thanks

**Reply by blakeblackshear** [reply]
The integration does not run frigate itself. It integrates frigate into home assistant. Same concept as the philips hue integration. The integration doesn't run your philips hue network, it just integrates the hub you already have.

You will need to run frigate so there is something to integrate home assistant with. You can run frigate as an addon in home assistant or on a separate computer with docker. The Coral needs to be plugged into whatever computer is running frigate.

**Reply by heapxor** [reply]
ah okay i see; 
but i assume https://github.com/blakeblackshear/frigate-hass-integration will install Frigate + integrate it with the HASS, correct?

**Reply by blakeblackshear** [reply]
No. It will only integrate Frigate. The addon will run it or you can run it in docker directly.

**Reply by heapxor** [reply]
okay, addon will run it itself and integrate with HASS (sorry got lost a bit). thanks
